packages/astro/src/index.ts

```diff
@@ -42,12 +42,16 @@ import "@keystatic/core/ui";
           vite,
         });
         injectRoute({
+          // @ts-ignore
           entryPoint: '@keystatic/astro/internal/keystatic-astro-page.astro',
```

Comment:
Is leaving this here to support Astro 3 as well?

Reply:
Yes